modules/containers-signature-verify-enable.adoc

```diff
@@ -78,19 +79,97 @@ $ butane 51-worker-rh-registry-trust.bu -o 51-worker-rh-registry-trust.yaml
 $ oc apply -f 51-worker-rh-registry-trust.yaml
 ----
 
-. Create a Butane config and corresponding machine config which writes the configuration to disk on the control plane nodes:
+. Apply the control plane machine config to the cluster:
 +
 [source,terminal]
 ----
-$ sed -e 's,\(machineconfiguration.openshift.io/role: \)worker,\1master,' \
-    -e 's,\(name: 51-\)worker,\1master,' 51-worker-rh-registry-trust.bu \
-    > 51-master-rh-registry-trust.bu
-$ butane 51-master-rh-registry-trust.bu -o 51-master-rh-registry-trust.yaml
+$ oc apply -f 51-master-rh-registry-trust.yaml
```

Comment:
We should delete above 2 lines "Apply the control plane machine config to the cluster:" and "$ oc apply -f 51-master-rh-registry-trust.yaml".  Because the master  node is enabled image signature verification by default.